index.js

```diff
@@ -119,6 +119,13 @@ Shopify.prototype.request = function request(uri, method, key, data, headers) {
     timeout: this.options.timeout,
     responseType: 'json',
     retry: 0,
+    hooks: {
+      beforeRedirect: [
+        (options) => {
+          options.url.href = `${options.url.protocol}//${this.options.apiKey}:${this.options.password}@${options.url.hostname}${options.url.pathname}`;
```

Comment:
What happens if the `Shopify` instance is created using the `accessToken` option instead of the `apiKey` and `password` options?